kordac/processors/errors/Error.py

```diff
@@ -0,0 +1,5 @@
+def Error(Exception):
+    """Base class for Errors.
+    (Exceptions from external sources such as inputs).
+    """
+    pass
```

Comment:
I think there is a newline, GitHub shows a little red symbol if there is no newline at the end of a file.